File: app/dbmodels/DeviceMemoryItem.php
<?php

include "_commondeviceitems.php";

$table['model'] = 'DeviceMemoryItem';

$table['fields']['device_memory_type_id'] = DBModels::type('integer');
$table['fields']['size_default']        = DBModels::type('integer');
$table['fields']['frequence']           = DBModels::type('string');

$table['relationships']['device_memory_type'] = array(
                                                'type' => 'belongsTo',
                                                'item' => 'DeviceMemoryType');

File: fusionglpi/database/device_memory_item.py
from fusionglpi import db

class DeviceMemoryItem(db.Model):
    id = db.Column(db.Integer, primary_key=True)
    entity_id = db.Column(db.Integer, db.ForeignKey('entity.id'))
    is_recursive = db.Column(db.Boolean, nullable=False, server_default="0")
    name = db.Column(db.String(255, None, True))
    manufacturer_id = db.Column(db.Integer, db.ForeignKey('manufacturer.id'))
    comment = db.Column(db.Text)
    device_memory_type_id = db.Column(db.Integer, db.ForeignKey('device_memory_type.id'))
    size_default = db.Column(db.Integer, nullable=False, server_default="0")
    frequence = db.Column(db.String(255, None, True))
